core/src/main/java/io/questdb/cutlass/http/HttpConnectionContext.java

```diff
@@ -346,17 +361,21 @@ public HttpConnectionContext of(long fd, @NotNull IODispatcher<HttpConnectionCon
     }
 
     public HttpRequestProcessor rejectRequest(int code, CharSequence userMessage) {
-        return rejectRequest(code, userMessage, null, null);
+        return rejectRequest(code, userMessage, false);
+    }
+
+    public HttpRequestProcessor rejectRequest(int code, CharSequence userMessage, boolean shutdownWrite) {
+        return rejectProcessor.rejectRequest(code, userMessage, null, null, AUTH_TYPE_NONE, shutdownWrite);
     }
 
     public HttpRequestProcessor rejectRequest(int code, byte authenticationType) {
         LOG.error().$("rejecting request [code=").$(code).I$();
-        return rejectProcessor.rejectRequest(code, null, null, null, authenticationType);
+        return rejectProcessor.rejectRequest(code, null, null, null, authenticationType, false);
     }
 
     public HttpRequestProcessor rejectRequest(int code, CharSequence userMessage, CharSequence cookieName, CharSequence cookieValue) {
```

Comment:
can we cascade these methods (that otherwise look very similar) so that we don't have an unused code in OSS?